src/Altinn.Common/Altinn.Common.PEP/Altinn.Common.PEP/Helpers/DecisionHelper.cs

```diff
@@ -83,12 +84,20 @@ private static XacmlJsonCategory CreateActionCategory(string actionType)
             return actionAttributes;
         }
 
-        private static XacmlJsonCategory CreateResourceCategory(string org, string app, string instanceOwnerId)
+        private static XacmlJsonCategory CreateResourceCategory(string org, string app, string instanceOwnerId, string instanceGuid)
```

Comment:
mix and match of instanceOwnerId and instanceOwnerPartyId. Use one of them consistently.

Reply:
renamed all to instanceOwnerPartyId